python/cugraph/cugraph/tests/dask/test_mg_doctests.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import contextlib
+import doctest
+import inspect
+import io
+import os
+import pathlib
+
+import numpy as np
+import pandas as pd
+import scipy
+import pytest
+
+import cugraph
+import cudf
+
+from dask.distributed import Client
+from dask_cuda import LocalCUDACluster
+import cugraph.comms as Comms
+
+datasets = pathlib.Path(cugraph.__path__[0]).parent.parent.parent / "datasets"
```

Comment:
I am a bit concern this will break if the structure of the cuGraph dir changes again. Perhaps we can still look for the env `RAPIDS_DATASET_ROOT_DIR` and then have `pathlib.Path(cugraph.__path__[0]).parent.parent.parent / "datasets"` as a default option?

Reply:
oops, I missed this in my review: yes, can we use the standard `RAPIDS_DATASET_ROOT_DIR` var here?